data/lang/module-cargorun/en.json

```diff
@@ -0,0 +1,610 @@
+      "description" : "",
+      "message" : "Danger:"
+   },
+   "DEADLINE" : {
+      "description" : "",
+      "message" : "Deadline:"
+   },
+   "DELIVER_TO" : {
+      "description" : "",
+      "message" : "Deliver to …"
+   },
+   "DISTANCE" : {
+      "description" : "",
+      "message" : "Distance:"
+   },
+   "PICKUP_FROM" : {
+      "description" : "",
+      "message" : "Pickup cargo from …"
+   },
+   "DENY_1" : {
+      "description" : "",
+      "message" : "I'm sorry, but this cargo is too valuable for me to hand them over to an inexperienced pilot."
+   },
+   "DENY_2" : {
+      "description" : "",
+      "message" : "Who are you? I do not know you and I do not trust you."
+   },
+   "DENY_3" : {
+      "description" : "",
+      "message" : "I'm sorry, I'm looking for an experienced pilot."
+   },
+   "DENY_4" : {
+      "description" : "",
+      "message" : "Excuse me, but I do not think that you can get the job done."
+   },
+   "DENY_5" : {
+      "description" : "",
+      "message" : "Don't waste my time! You are too wet behind the ears."
+   },
+   "DENY_6" : {
+      "description" : "",
+      "message" : "You're kidding, right? Come back when you're more experienced!"
+   },
+   "FAILUREMSG_1" : {
+      "description" : "",
+      "message" : "I'm very disappointed by the late delivery. You will not be paid."
+   },
+   "FAILUREMSG_2" : {
+      "description" : "",
+      "message" : "What should I do with these things? You are too late! The goods are now worthless!"
+   },
+   "FAILUREMSG_3" : {
+      "description" : "",
+      "message" : "Listen, we were clear about the deadline. I refuse you to pay."
+   },
+   "FAILUREMSG_4" : {
+      "description" : "",
+      "message" : "I'll make a note of your vessel identification number. From now on, I will reject all requests for work from you."
+   },
+   "FAILUREMSG__7__1" : {
+      "description" : "",
+      "message" : "You bastard, you have waited too long for this delivery. All my fluffy pets are starved!"
+   },
+   "WHAT_IS_THIS" : {
+      "description" : "",
+      "message" : "What's this? You have jettisoned my cargo? Are you crazy?"
+   },
+   "I_HAVE_DEBITED_YOUR_ACCOUNT" : {
+      "description" : "",
+      "message" : "I've debited your account with the missing goods."
+   },
+   "HOW_MUCH_MASS" : {
+      "description" : "",
+      "message" : "How much mass?"
+   },
+   "HOWMUCH_1" : {
+      "description" : "",
+      "message" : "{cargo} metric tons."
+   },
+   "HOWMUCH_2" : {
+      "description" : "",
+      "message" : "{cargo}t"
+   },
+   "HOWMUCH_3" : {
+      "description" : "",
+      "message" : "The mass is {cargo}t."
+   },
+   "HOWMUCH_4" : {
+      "description" : "",
+      "message" : "There are {cargo}t."
+   },
+   "HOWMUCH_5" : {
+      "description" : "",
+      "message" : "There are only {cargo}t."
+   },
+   "HOWMUCH_6" : {
+      "description" : "",
+      "message" : "Only {cargo}t."
+   },
+   "HOWMUCH_WHOLESALER_1" : {
```

Comment:
Could you please add to the translation description of `HOWMUCH_5` and `HOWMUCH_6` that these strings are only used for 1-2 items/tonnes of goods.